ensemble-dss/src/main/java/hec/dss/ensemble/DssDatabase.java

```diff
@@ -0,0 +1,125 @@
+package hec.dss.ensemble;
+
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+import hec.EnsembleDatabase;
+import hec.RecordIdentifier;
+import hec.ensemble.Ensemble;
+import hec.ensemble.EnsembleTimeSeries;
+import hec.heclib.dss.HecDss;
+import hec.io.TimeSeriesCollectionContainer;
+import hec.io.TimeSeriesContainer;
+
+
+/**
+ * DssDatabase implements EnsembleDatabase.
+ * It is used to work with time-series of ensembles.
+ * Data is stored in a DSS file, using the Collections and Version features of the DSS F-Part tag.
+ */
+public class DssDatabase implements EnsembleDatabase{
+    private String dssFileName;
+    public DssDatabase(String dssFileName){
+        this.dssFileName= dssFileName;
+    }
+
+    private static String buildDssPath(RecordIdentifier recordID, java.time.ZonedDateTime issue_time){
+        // to do define translation
+        return "//"+recordID.location+"/"+recordID.parameter+"///"+buildFpart(issue_time);
```

Comment:
i think we need to add in the start date time in the E part, and the timestep in the d part? or did i swap those? i dont remember the convention.

Both should be in the ensemble metadata - so we should have access to them

Reply:
Yes, buildDssPath(...)   is a work in progress.   We may need to have F part tags [ C: T: and V: ] for completeness.
Start time of data could be in T:  ,   issue date could be in V:  

D part also has the date, but we should ignore that as far as meta data, because it can end up being a block start date. 
E part is hardcoded right now as 1Hour, need to fix that.